CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Changelog
 
 ### Features
 
+- Implement a switch instead of a link for “hide used” addresses on the Receive screen ([PR 935](https://github.com/input-output-hk/daedalus/pull/935))
```

Comment:
@daniloprates please use "Implemented" instead of "Implement". This is the preferred form for CHANGELOG entries.
Also please make sure to use normal double-quotes `""` instead of `“”`.